zen_queries/template_response.py

```diff
@@ -5,10 +5,10 @@
 from zen_queries import queries_disabled
 
 
-class RenderMixin(object):
-    def render(self, *args, **kwargs):
+class RenderMixin:
+    def render(self) -> DjangoSimpleTemplateResponse:
```

Comment:
I wanted to make this `def render(self: T) -> T:`, but that isn't compatible with django-stubs where both `SimpleTemplateResponse` and `TemplateResponse` returns a literal `SimpleTemplateResponse` from `render()`. Also, the `T` type variable couldn't be bound to `SimpleTemplateResponse`, as `self: T` would then require `RenderMixin` to be a subclass of `SimpleTemplateResponse`, which it isn't. In short, support for typing mixins strictly isn't very good in Python 3.6. Once the required version is 3.8, one might attempt to type this more correctly using `typing.Protocol`.